generators/client/templates/angular/package.json.ejs

```diff
@@ -63,7 +63,7 @@
     "@angular/cli": "8.3.20",
     "@angular/compiler-cli": "8.2.14",
     "@ngtools/webpack": "8.3.20",
-    "@openapitools/openapi-generator-cli": "0.0.14-4.0.2",
+    "@openapitools/openapi-generator-cli": "1.0.8-4.2.2",
```

Comment:
We released @openapitools/openapi-generator-cli `1.0.10-4.2.3` a few days ago: https://www.npmjs.com/package/@openapitools/openapi-generator-cli